### PR TITLE
[DMU] Implemented all currently previewed cards

### PR DESCRIPTION
**Pull request by theelk801**
Let me know how many mistakes I made

### CODE REVIEW

File: Mage.Sets/src/mage/cards/e/EvolvedSleeper.java
Comment (Alex-Vasile) [inline Mage.Sets/src/mage/cards/e/EvolvedSleeper.java:120]: Are there instances where the player is going to be null but the creature is not?

File: Mage.Sets/src/mage/cards/e/EvolvedSleeper.java
Comment (Alex-Vasile) [inline Mage.Sets/src/mage/cards/e/EvolvedSleeper.java:96]: Is this supposed to happen as one effect since the card text has a comma between the +1/+1 gain and the draw? Or should it be split over two so that anything that cares about the draw or the life-loss sees the evolved sleeper as already having the +1/+1?

File: Mage.Sets/src/mage/cards/s/ShivanDevastator.java
Comment (Alex-Vasile) [inline Mage.Sets/src/mage/cards/s/ShivanDevastator.java:33]: append with comma?

File: Mage.Sets/src/mage/sets/DominariaUnited.java
Comment (Alex-Vasile) [inline Mage.Sets/src/mage/sets/DominariaUnited.java:19]: should be DMU

File: Mage/src/main/java/mage/game/command/emblems/JayaFieryNegotiatorEmblem.java
Comment (Alex-Vasile) [inline Mage/src/main/java/mage/game/command/emblems/JayaFieryNegotiatorEmblem.java:60]: This doesn't seem to check any of the conditions that the spell should have (red instant or sorcery).

Comment (theelk801) [reply Mage/src/main/java/mage/game/command/emblems/JayaFieryNegotiatorEmblem.java:60]: it doesn't need to, that's handled by the trigger

File: Mage.Sets/src/mage/sets/DominariaUnited.java
Comment (theelk801) [reply Mage.Sets/src/mage/sets/DominariaUnited.java:19]: whoops, good call

File: Mage.Sets/src/mage/cards/s/ShivanDevastator.java
Comment (theelk801) [reply Mage.Sets/src/mage/cards/s/ShivanDevastator.java:33]: was this a new feature that was added while I was gone?

File: Mage.Sets/src/mage/cards/e/EvolvedSleeper.java
Comment (theelk801) [reply Mage.Sets/src/mage/cards/e/EvolvedSleeper.java:96]: why would it matter whether it has the counter?

File: Mage/src/main/java/mage/game/command/emblems/JayaFieryNegotiatorEmblem.java
Comment (Alex-Vasile) [reply Mage/src/main/java/mage/game/command/emblems/JayaFieryNegotiatorEmblem.java:60]: Got you. I missed the filter

File: Mage.Sets/src/mage/cards/e/EvolvedSleeper.java
Comment (Alex-Vasile) [reply Mage.Sets/src/mage/cards/e/EvolvedSleeper.java:96]: I fixed one or two issues like this. See the checked off one https://github.com/magefree/mage/issues/6761. There was another but can't find it atm. The second part of the effect (draw and lose life) may interact differently with other cards if there is a creature with a +1/+1 counter on the board. Having them as one effect will cause the +1/+1 gain and the draw+lose life to occur simultaneously rather than sequentially.

Comment (theelk801) [reply Mage.Sets/src/mage/cards/e/EvolvedSleeper.java:96]: oh right, any objection to adding `game.applyEffects()` in between?

File: Mage.Sets/src/mage/cards/s/ShivanDevastator.java
Comment (Alex-Vasile) [reply Mage.Sets/src/mage/cards/s/ShivanDevastator.java:33]: Sorry I was thinking of `.concatBy()`, but that's only for effects.

File: Mage.Sets/src/mage/cards/e/EvolvedSleeper.java
Comment (Alex-Vasile) [reply Mage.Sets/src/mage/cards/e/EvolvedSleeper.java:96]: > oh right, any objection to adding `game.applyEffects()` in between?

I can't comment one way or another. For the linked issue I split the effect across two. Not sure of the implication of `game.applyEffects()`.

Comment (theelk801) [reply Mage.Sets/src/mage/cards/e/EvolvedSleeper.java:96]: the only way I can see this mattering is if the creature's P/T with the counter is relevant, which is what `game.applyEffects()` will do, as the creature should show as having a counter by the time the draw happens anyway

Comment (Alex-Vasile) [reply Mage.Sets/src/mage/cards/e/EvolvedSleeper.java:96]: I think it should be added to be technically correct.

Even if there isn't a card right now that cares about it, there might be one in the future.

Comment (theelk801) [reply Mage.Sets/src/mage/cards/e/EvolvedSleeper.java:96]: I don't disagree, I'll add it in

**Comment by Alex-Vasile** [comment]
@JayDi85 is there a plan to release another stable version of the game, or are we only doing beta releases from now on?

If we're doing just the beta, then I'd like to merge this once everything has been addressed.

If we're going to continue doing stable releases I'd like to know what we're waiting for. (And I think we should hold off on merging this or any new sets until the stable version) so that we can focus on finishing off the sets currently in progress and on fixing existing bugs.

**Review by Alex-Vasile — APPROVED** [review]

**Comment by Alex-Vasile** [comment]
I'd still wait a bit to give JayDi a chance to reply.

**Comment by Alex-Vasile** [comment]
> @JayDi85 is there a plan to release another stable version of the game, or are we only doing beta releases from now on?
> 
> If we're doing just the beta, then I'd like to merge this once everything has been addressed.
> 
> If we're going to continue doing stable releases I'd like to know what we're waiting for. (And I think we should hold off on merging this or any new sets until the stable version) so that we can focus on finishing off the sets currently in progress and on fixing existing bugs.

@JayDi85? Can we please get some communication on the direction of the project

File: Mage.Sets/src/mage/cards/e/EvolvedSleeper.java
Comment (Alex-Vasile) [reply Mage.Sets/src/mage/cards/e/EvolvedSleeper.java:96]: Can you please add the apply.Effect()

Comment (theelk801) [reply Mage.Sets/src/mage/cards/e/EvolvedSleeper.java:96]: I did

Comment (Alex-Vasile) [reply Mage.Sets/src/mage/cards/e/EvolvedSleeper.java:96]: My bad. I missed it since this didn't my show as being outdated or resolved.

**Comment by Alex-Vasile** [comment]
@JayDi85

**Pull request merged**
